README.md

```diff
@@ -1,11 +1,13 @@
 # pretty-json
-![version](https://img.shields.io/badge/dynamic/json.svg?url=https://raw.githubusercontent.com/SlimIO/Pretty-JSON/master/package.json?token=AOgWw3vrgQuu-U4fz1c7yYZyc7XJPNtrks5catjdwA%3D%3D&query=$.version&label=Version)
-[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/SlimIO/Pretty-JSON/commit-activity)
-![MIT](https://img.shields.io/github/license/mashape/apistatus.svg)
-![dep](https://img.shields.io/david/SlimIO/Pretty-JSON)
-![size](https://img.shields.io/bundlephobia/min/@slimio/pretty-json)
-[![Known Vulnerabilities](https://snyk.io//test/github/SlimIO/Pretty-JSON/badge.svg?targetFile=package.json)](https://snyk.io//test/github/SlimIO/Pretty-JSON?targetFile=package.json)
-[![Build Status](https://travis-ci.com/SlimIO/Pretty-JSON.svg?branch=master)](https://travis-ci.com/SlimIO/Pretty-JSON)
+![version](https://img.shields.io/badge/dynamic/json.svg?style=for-the-badge&url=https://raw.githubusercontent.com/TopCli/Pretty-JSON/master/package.json&query=$.version&label=Version)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg?style=for-the-badge)](https://github.com/TopCli/Pretty-JSON/commit-activity)
+[![mit](https://img.shields.io/github/license/Naereen/StrapDown.js.svg?style=for-the-badge)](https://github.com/TopCli/Pretty-JSON/blob/master/LICENSE)
```

Comment:
```suggestion
[![mit](https://img.shields.io/badge/License-MIT-blue.svg?style=for-the-badge)](https://github.com/TopCli/Pretty-JSON/blob/master/LICENSE)
```
In all packages we use `Naereen/StrapDown.js` repository license badge, wouldn't be better to use this one instead ? 
[![mit](https://img.shields.io/badge/License-MIT-blue.svg?style=for-the-badge)](https://github.com/TopCli/Pretty-JSON/blob/master/LICENSE)